examples/all_in_one/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DOCKER=${DOCKER:-docker}
+DOCKER_COMPOSE=${DOCKER_COMPOSE:-docker-compose}
+
+echo
+
+for req in DOCKER DOCKER_COMPOSE; do
+    if ! ${!req} -v >/dev/null 2>1 ; then
+        echo "Unable to run ${!req}, make sure that it's installed and working."
+        exit 1
+    fi
+done
+
+
+${DOCKER_COMPOSE} down -v
+${DOCKER_COMPOSE} up -d
+
+echo "---------------------------------------------------------------------------"
+echo "All services started, you can access them at the following addresses:"
+echo "Prometheus: http://localhost:9090/"
+echo "Grafana: http://localhost:3000"
+echo " User: admin, Password: admin"
```

Comment:
I'd personally prefer if this info would be in some README.md of this example along with some description.

Reply:
README with the overall description of the setup is yet to be written, this is why this PR is marked as WIP. :)

Reply:
Rather simple README added within https://github.com/seznam/slo-exporter/pull/4/commits/51f427078c901477bd4737f900190c95585ebdc1

Not sure whether it's enough, though. Let me know if you think that something important is missing.

Reply:
> I would prefer to use at least two classes. The concept of slo_class is not understandable it there is only one class here.

Current configuration is rather minimalist, while I believe that it should be clear to any user how to add new classes if needed.

I would prefer to keep it as it is now, and eventually iterate on that in separate in PR.